docs/function-execution.md

```diff
@@ -35,6 +35,28 @@ True # returns False if registration fails, e.g., if one of the referenced funct
 * DAGs can have arbitrary branches and connections and have multiple sources, but there must be only one sink function in the DAG. The result of this sink function is what is returned to the caller.
 * For those familiar with the Anna KVS, all use of lattices is abstracted away from the Droplet user. The serialization and deserialization is done automatically by the runtime, and only Python values are passed into and out of all API functions.
 
+## Registering and Executing Classes
+
+Some functions require preinitialization that can be potentially expensive (e.g., loading a machine learning model).
+Instead of repeating this expensive initialization process on every request, users can also register a Python class with the runtime instead of a function.
+This class will be initialized as a part of a DAG<sup>1</sup>, and the initialized state will persist until the resources are deallocated.
```

Comment:
Suggestion: Change to "`When this class is initialized as part of a DAG, the initialized state will persist until the resources are deallocated`"?

Reply:
Changed.